libnmstate/nm/applier.py

```diff
@@ -170,17 +169,17 @@ def set_ifaces_admin_state(ifaces_desired_state, con_profiles=()):
                 InterfaceState.ABSENT,
             ):
                 nmdevs = _get_affected_devices(iface_desired_state)
-                for nmdev in nmdevs:
-                    remove_devs_actions[nmdev] = [
+                for affected_nmdev in nmdevs:
+                    remove_devs_actions[affected_nmdev] = [
                         device.deactivate,
                         device.delete,
                     ]
-                    if nmdev.get_device_type() in (
-                        nmclient.NM.DeviceType.OVS_BRIDGE,
-                        nmclient.NM.DeviceType.OVS_PORT,
-                        nmclient.NM.DeviceType.OVS_INTERFACE,
-                    ):
-                        remove_devs_actions[nmdev].append(device.delete_device)
+                if (
```

Comment:
It would be good to have a test that shows that this is working for example by creating an interface with `ip link`, configuring it with Nmstate and then removing the interface with Nmstate.

Reply:
Done